### PR TITLE
Fix/provider consolelog

### PR DESCRIPTION
**Pull request by chrismahoney**
- Removes provider info console logging from testing #251.
- Fixes the missing "key" attribute for React fragments in Editor panel, which removes a console log error. 
- Explicitly sets <> </> tags to React.Fragment elements.

### CODE REVIEW

**Comment by chrismahoney** [comment]
@coleam00 could you look this over for me when you get a sec? Just simple cleanup of console logging to remove an error and extra log entries.

File: app/components/chat/BaseChat.tsx [hunk 1]
```diff
@@ -111,12 +111,10 @@ export const BaseChat = React.forwardRef<HTMLDivElement, BaseChatProps>(
     },
     ref,
   ) => {
-    console.log(provider);
```
Comment (chrismahoney) [inline app/components/chat/BaseChat.tsx:114]: Here's the removal of the provider logs

**Comment by wonderwhy-er** [comment]
I just pulled and checked that it works.
Merging.

**Comment by coleam00** [comment]
Thanks @wonderwhy-er, sorry I missed this notification @chrismahoney!

**Pull request merged**